source/appendices/e-audit-requirements-and-tests/03-denial-tests.rst

```diff
@@ -0,0 +1,45 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.3 Audit Tests for Denials
+---------------------------
+
+Report providers operating platforms where turnaways or denials are in operation MUST be subject to the following audit tests for denials. For report providers operating multiple platforms, the audit scope as defined in :numref:`audit` MUST include platforms where turnaways or denials are in operation. Where either Limit_Exceeded or No_License denials do not apply to a report provider, auditors MUST note this in the audit report. This does not require an exemption from the COUNTER Project Director.
+
+These audit tests should represent the scope of the platform under audit. That is, where a platform is made up of a mixture of content with Data_Types Article, Multimedia and Patent, the auditor SHOULD represent each of those Data_Types proportionately in the audit test.
+
+These audit tests apply to the Database Report, Title Report and Item Report. The DR_D2, TR_B2, and TR_J2 Standard Views of COUNTER Reports will be deemed COUNTER-compliant if the metrics match those in the relevant COUNTER Report, with the appropriate aggregation.
+
+
+E.3.1 Limit_Exceeded
+""""""""""""""""""""""
+Note that the account used for this testing MUST have concurrent / simultaneous-user limit (concurrency limits) set at a single user. A second user attempting to access the database would be denied.
+
+**Option 1**: The report provider denies the user access when the concurrency limit is exceeded upon login.
```

Comment:
As mentioned Limit_Exceeded is only defined for databases and items, not for platforms, so there is no way to include these denials in a report. I noticed that the introduction only mentions DR, TR and IR, so maybe this option should be removed?

Reply:
I have checked this one with the auditors and they say it's a common use case, despite not being reflected in the PR, so should be retained